db/schema.rb

```diff
@@ -31,8 +31,15 @@
   add_index "admins", ["email"], name: "index_admins_on_email", unique: true
   add_index "admins", ["reset_password_token"], name: "index_admins_on_reset_password_token", unique: true
 
+  create_table "attachments", force: true do |t|
+    t.integer "source_id"
+    t.integer "asset_id"
+    t.string  "asset_type"
+  end
+
+  add_index "attachments", ["source_id"], name: "index_attachments_on_source_id"
```

Comment:
Is there a way in Rails migrations to have a compound index on three columns? That would be ideal here.